pkg/adapter/awscloudwatchlogsource/adapter.go

```diff
@@ -80,33 +74,31 @@ func NewAdapter(ctx context.Context, envAcc pkgadapter.EnvConfigAccessor, ceClie
 	logger := logging.FromContext(ctx)
 
 	env := envAcc.(*envConfig)
-	a := common.MustParseARN(env.ARN)
 
-	awsCfg := aws.NewConfig()
-	awsCfg.WithCredentials(credentials.NewStaticCredentials(env.AccessKey, env.SecretKey, ""))
-	cfg := session.Must(session.NewSession(awsCfg.
-		WithRegion(a.Region).
-		WithMaxRetries(5),
+	arn := common.MustParseARN(env.ARN)
```

Comment:
This is the one area where I've been given warning about the package name coinciding with the variable name, which is why I stuck with `a` instead of `arn`.

Reply:
Where do you see this warning?

Reply:
Reverted in cf58977f173705371da6b364f3c2cb6292ca4c6e

Reply:
Feature of the IDE I'm using: https://www.jetbrains.com/help/go/code-inspections-in-go.html#Probable_bugs


Imported package name as name identifier | Reports declarations of variables, arguments or functions that overlap with the used import.                     While  legal, such declarations will make using the package exported  identifiers impossible after the declaration or create confusion when  reading the code. | Warning
-- | -- | --